src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -150,6 +151,44 @@ public static string GetDistroVersionString()
             }
         }
 
+        private static Lazy<Version> m_icuVersion = new Lazy<Version>(GetICUVersion);
+        public static Version ICUVersion => m_icuVersion.Value;
+
+        public static bool ShouldUseIcu => ICUVersion > new Version(0,0,0,0);
```

Comment:
Nit: No other property on PlatformDetection is called ShouldXXX.

Would `IsIcuGlobalization` be a better name for this?

Reply:
Yeah, that's a good suggestion.